src/iso19111/c_api.cpp

```diff
@@ -289,7 +289,6 @@ PJ_OBJ_LIST::~PJ_OBJ_LIST() = default;
  * @param ctx Ignored
  * @param autoclose Ignored
  * @since 6.2
- * @deprecated Since 8.1
```

Comment:
Please keep it as a regular Doxygen warning then, without the '@'

Reply:
Alternatively, does PROJ have a mechanism to mark things deprecated?  For example, is the [C++14 `[[deprecated]]`](https://en.cppreference.com/w/cpp/language/attributes/deprecated) too new?